python/_nimblephysics/constraint/ConstraintSolver.cpp

```diff
@@ -200,7 +201,16 @@ void ConstraintSolver(py::module& m)
       .def(
           "solve",
           +[](dart::constraint::ConstraintSolver* self,
-              dart::simulation::World* world) { self->solve(world); });
+              dart::simulation::World* world) { self->solve(world); })
+      .def(
+          "lcpSolveCallback",
+          +[](dart::constraint::ConstraintSolver* self,
+              dart::simulation::World* world) {
```

Comment:
I suspect you just forgot to change this signature. You don't need the `world` argument anymore, no?